bindings/nodejs/types/transactionOptions.ts

```diff
@@ -34,6 +34,14 @@ export type CustomAddress = {
     value: string;
 };
 
+/** Native token options for minting more native tokens */
+export interface MintMoreNativeTokenOptions {
+    /** Hex encoded number */
+    additionalSupply: HexEncodedAmount;
+    /** Native token id */
+    tokenId: string;
+}
```

Comment:
Are we expecting more options? Otherwise these seem like very basic parameters that maybe should be part of the function signature?

Reply:
Currently I don't expect more, one possible thing one could add would be `Option<String>` for an address to which the new minted tokens should go, but not sure if anyone cares about that

Reply:
I'm fine keeping the options if we can already think of some, just saying that additionalSupply and tokenId are mandatory parameters and not really options 🤔

Reply:
Ah right, should I just change the name then?

Reply:
They are mandatory so they should be method parameters and not even in a struct IMO

Reply:
The supply fields in NativeTokenOptions are also mandatory, should they also be parameters? But having the different options split feels also weird

Reply:
Yeah but mandatory parameters are NOT options? To me, anything ending in `Options` should only contains `Option`s, mandatory things should be part of the API, I think it's clearer that way.

Reply:
You also have the option to set something to true or false, which could be mandatory, not optional, but still an option :trollface:

Reply:
But I'm fine with changing it to move not optional things to parameters

Reply:
If we do that, do you think we should keep the options even if empty at the moment just to avoid any future breaking change?

Reply:
Would still be a breaking change, no?

Reply:
If the members are private and we provide new and getters it should it should be fine ? Overall I think we have way too many structs with public fields, I think we should change that before the release.

Reply:
Ah yeah if they're private it should work

Reply:
@thibault-martinez is it now like you imagined it?

Reply:
Yeah, LGTM